base/src/main/java/io/spine/validate/AbstractValidatingBuilder.java

```diff
@@ -286,8 +286,8 @@ private static boolean isSetOnce(FieldDescriptor field) {
     private void logError(FieldDescriptor field) {
         String fieldName = field.getFullName();
         _error("Error found in `%s`. " +
-                    "Repeated and map fields can't be marked as `(set_once) = true`",
-                    fieldName);
+               "Repeated and map fields can't be marked as `(set_once) = true`.",
```

Comment:
Please replace `can't` according to [these recommendations](https://www.grammarly.com/blog/cannot-or-can-not/).